Components/Avatar/avatar.css

```diff
@@ -0,0 +1,75 @@
+@import url("/Util/util.css");
+
+.avatar {
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+    margin: 0 0.5rem;
```

Comment:
It's better to follow a particular sizing unit inside a class or inside an attribute. `rem` makes things more responsive whereas `px` doesn't.